### PR TITLE
Add badge for first onboarding visitors

### PR DESCRIPTION
**Pull request by csubira**
Add badge for **first Onboarding visitors** (removed once the onboarding wizard has been opened):

Screen Shot 2019-04-17 at 13 40 34

> 

This PR includes as well:
- Fix hover in Selectors card
- Fix icon path in fake Dataset card of Builder tutorial
- Increase border radius in selector tags to avoid button confusion

### CODE REVIEW

**Comment by kukukaka** [comment]
Nice. Thanks @csubira!

File: lib/assets/javascripts/new-dashboard/components/Onboarding/distributor/Selector.vue [hunk 1]
```diff
@@ -86,18 +91,26 @@ export default {
...
       padding: 6px 16px;
       border: 1px solid $onboarding-tags__border-color;
-      border-radius: 3px;
+      border-radius: 18px;
       font-size: 10px;
       text-transform: uppercase;
     }
   }
+
+  &:hover {
```
Comment (alejandraarri) [inline lib/assets/javascripts/new-dashboard/components/Onboarding/distributor/Selector.vue]: I would also add these styles to :focus state and remove the default browser outline.

**Comment by alonsogarciapablo** [comment]
Usage of Onboarding is going to skyrocket once we merge this 😉!

**Comment by alejandraarri** [comment]
Acceptance done! Tested with different users. 👍

**Comment by csubira** [comment]
retest this please

**Pull request merged**